package-lock.json

```diff
@@ -32,7 +32,7 @@
         "stylelint-config-standard": "^24.0.0"
       },
       "engines": {
-        "node": ">=16.3.0"
+        "node": ">=16.3.0 < 22.4.0"
```

Comment:
After discussion amongst team, please can we remove the upper limit? One less thing to have to keep maintaining